hollow/src/main/java/com/netflix/hollow/api/producer/AbstractHollowProducer.java

```diff
@@ -555,6 +558,7 @@ void publish(ProducerListeners listeners, long toVersion, Artifacts artifacts) t
                 artifacts.reverseDelta = stageBlob(listeners,
                         blobStager.openReverseDelta(toVersion, readStates.current().getVersion()));
 
+                publishHeaderBlob(artifacts.header);
```

Comment:
nit: this call can be moved up before the if-else, since it is duplicated in both if and else

Reply:
ack